client/src/utils/utils.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+export function makeQueryStringTransformable (params: Object): Object {
```

Comment:
the reason why adding types here is difficult or not worthy, because its generic method about which we only know it accepts and object, we cant limit or bound it to accept specific types of object, it will return object with same keys but normalized values transformable to query-string.

Reply:
@isaadabbasi yes thats fine, but adding Object doesnt help anyone so there is no point of adding it there. Instead we should add a detailed comment explaining the inputs and outputs for this function.

Reply:
sure. that'd be better.